src/GNDStk/BodyText/test/BodyText.test.cpp

```diff
@@ -0,0 +1,93 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+using namespace njoy::GNDStk::core;
+
+SCENARIO("Testing GNDStk BodyText") {
+   // Most BodyText functionality is tested in the individual test files.
+   // There are just a few things we'll do here.
+
+   // Ensure that we can make const and non-const <true> and <false> BodyText
+   // objects. Note that BodyText has only a default constructor.
+
+   {
+      const BodyText<true> cbt;
+      CHECK(cbt.length   () == 0);
+      CHECK(cbt.size     () == 0);
+      CHECK(cbt.valueType() == "");
+      CHECK(cbt.string   () == "");
+   } {
+      const BodyText<false> cbf;
+      // no data for <false>
+   } {
+      BodyText<true> nbt;
+      CHECK(nbt.length   () == 0);
+      CHECK(nbt.size     () == 0);
+      CHECK(nbt.valueType() == "");
+      CHECK(nbt.string   () == "");
+   } {
+      BodyText<false> nbf;
+      // no data for <false>
+   }
```

Comment:
Weird is in the eye of the beholder. :-) But, yes, I can do this. Would you be fine if I just do these things for the next PR, which will include Component tests as well? Given that I've already tweaked things here and there in BodyText's code base, which of course is related to Component, this would be less messy than dealing wtih a merge.

Reply:
I've done like this in a handful of other test files from before, and can change it in those places as well, to be consistent.